src/Features/LanguageServer/Protocol/Extensions/ProtocolConversions.cs

```diff
@@ -146,7 +147,18 @@ public static Uri GetUriFromFilePath(string filePath)
             if (filePath is null)
                 throw new ArgumentNullException(nameof(filePath));
 
-            return new Uri(filePath, UriKind.Absolute);
+            if (Path.IsPathRooted(filePath))
```

Comment:
this was mainly for tests.  test documents are created as non-rooted paths (e.g. "test1.cs") so we can't use absolute paths.  If its non-rooted we just create a relative URI instead.

Reply:
why are we using URIs :D

Reply:
@CyrusNajmabadi - you mean in general for exposing the doc changes to intellicode?  technically we could use roslyn documents I suppose

@pgroene do you have a preference?

Reply:
confirmed offline intellicode can use documentid.  will update this to match

Reply:
can we revert this change then?